### ISSUE TITLE
link_to_remote

### ISSUE THREAD

**Issue by jrencz** [teleological/redmine_stealth#18]
this helper is removed in Rails 3.
File lib/redmine_menu_manager_extensions.rb was last time updated for 0.4.0
During install process on Redmine 2.0.3.devel I got error.

**Comment by xadh00m** [comment]
I´ve tested the stealth plugin on the latest redmine 1.2.1. The same error occurs here.

Other plugins suffer the same issues e.g. here:
https://bitbucket.org/akiko_pusu/redmine_issue_templates/issue/13/redmine-210-problems-link_to_remote 

A possible fix using "preview_link" instead is used here:
https://github.com/amartel/redmine_meetings/pull/3/files

Any chance that the plugin will become compatible soon?

**Comment by teleological** [comment]
Looks like this problem is due to Redmine 2.1 switching from Prototype to jQuery:

  http://www.redmine.org/issues/11445

Shouldn't be too hard to make it work with either. Looking at it now.

**Comment by teleological** [comment]
Fixed in v0.6.0

**Comment by R-i-c-k-y** [comment]
why did you close this issue? the code in v.0.6.0 (File lib/redmine_menu_manager_extensions.rb) i still

[...]
    module MenuHelper

```
  def render_single_menu_node_with_remote(item, caption, url, is_sel)
    if remote_options = item.remote_options
      non_html_options = remote_options.kind_of?(Hash) ?
        remote_options.reverse_merge(:url => url) : { :url => url }
        link_to_remote(h(caption),
                    non_html_options,
                    item.html_options(:selected => is_sel))
```

[...]
